ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/panelcontainer/v1/panelcontainer/saveasfragment_dialog/.content.xml

```diff
@@ -0,0 +1,84 @@
+          sling:resourceType="cq/gui/components/authoring/dialog"
+          extraClientlibs="[core.forms.components.panelcontainer.v1.editor]">
+    <content
+            granite:class="cmp-adaptiveform-saveasfragment"
+            jcr:primaryType="nt:unstructured"
+            jcr:title=""
+            sling:resourceType="granite/ui/components/coral/foundation/form/fieldset">
+        <items jcr:primaryType="nt:unstructured">
+            <name
+                jcr:primaryType="nt:unstructured"
+                sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                fieldLabel="Name"
+                required="{Boolean}true"
+                name="name">
+                <granite:data
+                        jcr:primaryType="nt:unstructured"
+                        node-name-validation=""/>
+            </name>
+            <title
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                    fieldLabel="Title"
+                    name="jcr:title">
+            </title>
+            <targetPath
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/form/pathfield"
+                    nodeTypes="nt:folder, sling:Folder, sling:OrderedFolder"
+                    filter="folder"
+                    fieldLabel="Target Path"
+                    name="targetPath"
+                    required="{Boolean}true"
+                    rootPath="/content/dam/formsanddocuments"/>
+            <templateselector
+                    granite:class="cmp-adaptiveform-panelcontainer__templateselector"
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/form/autocomplete"
+                    emptyText="Type here or browse"
+                    fieldLabel="Select Fragment Template"
+                    filter="template"
+                    name="templatePath"
+                    required="{Boolean}true">
+                <datasource
+                        jcr:primaryType="nt:unstructured"
+                        fdVersion="2"
+                        sling:resourceType="fd/fm/gui/components/admin/formelements/templatedataprovider"/>
+                <options
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/coral/foundation/form/autocomplete/list"/>
+                <values
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/coral/foundation/form/autocomplete/tags"/>
+            </templateselector>
+            <fragmentselectcontainer
+                jcr:primaryType="nt:unstructured"
+                granite:class="coral-Form-fieldwrapper cmp-adaptiveform-saveasfragment__fragmentselector-container"
```

Comment:
Why do you need this class, coral-Form-fieldwrapper

Reply:
Just to align with style of other field wrapper in dialog . This field is appended on client side while others are rendered at server. 
[1]: https://developer.adobe.com/experience-manager/reference-materials/6-5/coral-ui/coralui3/styles.html#coral-Form